automated install/basic-install.sh

```diff
@@ -2219,6 +2219,9 @@ FTLinstall() {
         ftlBranch="master"
     fi
 
+    local binary
+    binary="${1}"
```

Comment:
This probably should throw an error of some type if `$1` is empty. Either error here or pass a retval to the caller to signal failure.